config/service.yml

```diff
@@ -123,7 +123,7 @@ services:
     Invertus\SaferPay\Service\PaymentRestrictionValidation:
         class: Invertus\SaferPay\Service\PaymentRestrictionValidation
         arguments:
-            - !tagged saferpay.paymentrestriction
+            - [ '@Invertus\SaferPay\Service\PaymentRestrictionValidation\ApplePayPaymentRestrictionValidation', '@Invertus\SaferPay\Service\PaymentRestrictionValidation\BasePaymentRestrictionValidation', '@Invertus\SaferPay\Service\PaymentRestrictionValidation\KlarnaPaymentRestrictionValidation', '@Invertus\SaferPay\Service\PaymentRestrictionValidation\BasePaymentRestrictionValidation' ]
```

Comment:
maybe its better to create some kind of restriction provider? i m not sure just looks strange to use array like this here.